app/controllers/annotations_overview_controller.rb

```diff
@@ -0,0 +1,54 @@
+class AnnotationsOverviewController < ApplicationController
+  layout "application_no_sidebar_with_background"
+
+  def show
+    user_annotations = Annotation.where(user_id: current_user.id)
+                                 .map { |a| extract_relevant_information(a) }
+    @annotations_by_lecture = user_annotations.group_by { |annotation| annotation[:lecture] }
+    # TODO: don't include lecture key in the hash anymore after grouping
+    # Maybe we can even gruop in the SQL query directly?
+
+    @show_students_annotations = current_user.teachable_editor_or_teacher?
+    if @show_students_annotations
+      student_annotations =
+        Annotation.where(medium_id: medium_ids_for_teacher_or_editor,
+                         visible_for_teacher: true)
+                  .map { |a| extract_relevant_information(a, is_shared: true) }
+      @student_annotations_by_lecture = student_annotations.group_by do |annotation|
+        annotation[:lecture]
+      end
+    end
+
+    render "annotations/annotations_overview"
```

Comment:
If the action is renamed to `index` and moved into the `AnnotationsController`, and the view is renamed to `index.html.erb` then this line becomes obsolete.

Reply:
modified this line to read: `render layout: "application_no_sidebar_with_background"` since we want to use the custom layout here